app/code/Magento/InventoryConfigurableProduct/view/adminhtml/web/js/variations/steps/bulk.js

```diff
@@ -24,7 +26,29 @@ define([
 
         /** @inheritdoc */
         initialize: function () {
+            var self = this,
+                sections;
+
             this._super();
+
+            sections = this.sections();
+            sections.quantityPerSource = {
+                label: 'Quantity Per Source',
+                type: ko.observable('none'),
+                value: ko.observable(),
+                attribute: ko.observable()
+            };
+            this.sections(sections);
+
+            /**
+             * Make options sections.
+             */
+            this.makeOptionSections = function () {
```

Comment:
As I see, this function we use only one time. Please rewrite her without using `self` variable. You can bind context and return the new object.
``` 
this.makeOptionSelections = function(){...}.bind(this);
```
```
return {
       images: this.makeImages(null),
       price: this.price,
       quantityPerSource = this.quantityPerSource
}
```
 And use her without `new`, as a simple function instead of a constructor.

Reply:
Fixed.